Resources/Prototypes/_EinsteinEngines/Entities/Modsuits/security-responsory.yml

```
@@ -0,0 +1,160 @@
+  parent: ClothingModsuitGauntletsStandard
+  id: ClothingModsuitGauntletsResponsorySecurity
+  name: Security Responsory modsuit gauntlets
+  description: A pair of gauntlets made for the Security Responsory modsuit.
+  categories: [HideSpawnMenu]
+  components:
+    - type: Sprite
+      sprite: _EinsteinEngines/Clothing/NanoTrasen/Modsuits/security-responsory.rsi
+    - type: SealableClothingVisuals
+      visualLayers:
+        gloves:
+        - state: equipped-HAND-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)